config/locales/server.en.yml

```diff
@@ -1347,6 +1347,7 @@ en:
     s3_backup_bucket: "The remote bucket to hold backups. WARNING: Make sure it is a private bucket."
     s3_endpoint: "The endpoint can be modified to backup to an S3 compatible service like DigitalOcean Spaces or Minio. WARNING: Use default if using AWS S3"
     s3_force_path_style: "Enforce path-style addressing for your custom endpoint. IMPORTANT: Required for using Minio uploads and backups."
+    s3_configure_tombstone_policy: "Enable automatic policy configuration for Tombstone."
```

Comment:
help text can be a bit better "Enable automatic deletion policy configuration for tombstone uploads, without this policy no space will be reclaimed after uploads are deleted" ... something like that